src/transformers/trainer.py

```diff
@@ -182,6 +182,16 @@
 logger = logging.get_logger(__name__)
 
 
+def recursive_print(state_dict, prefix=""):
```

Comment:
if it's useful save in debug_utils.py?

Reply:
Oops, it's not supposed ot be in the PR anymore.